.github/workflows/ci.yml

```diff
@@ -23,18 +23,22 @@ jobs:
     steps:
       - name: Checkout repository
         uses: actions/checkout@v2
+      - name: Install linux-modules-extra-azure
+        run: |
+          sudo apt update
+          sudo apt install -y linux-modules-extra-azure
       - name: Install Rust
         uses: actions-rs/toolchain@v1
         with:
           toolchain: ${{ matrix.rust }}
           override: true
           profile: minimal
       - name: Build
-        run: make build CARGOFLAGS="--verbose"
+        run: CARGOFLAGS="--verbose" make build
```

Comment:
Please drop this chunk. Both approaches are equivalent and the original one is more canonical.

Reply:
redo with #85